EIPS/Increase block rewards to 5 ETH.md

```diff
@@ -0,0 +1,34 @@
+---
+eip: <to be assigned>
+title: Increase block rewards to 5 ETH
+author: Ben Tinner (@Terra854)
+discussions-to: https://www.reddit.com/r/ethereum/comments/k4252u/thoughts_about_increasing_block_rewards_till_we/
+status: Draft
+type: Standards Track
+category Core
+created: 2020-12-01
+---
+
+## Simple Summary
+This EIP will increase the block reward paid to proof-of-work (POW) miners, giving them more opporturnities to acquire the minimum number of ETH needed to run a standalone proof-of-stake (POS) validator node before full implementation of ETH 2.0
```

Comment:
```suggestion
Changes the block reward paid to proof-of-work (POW) miners to 5 ETH.
```

Reply:
Simple summary should just state in very few words what this EIP does.  It should not include motivation or reasoning.